docker-compose.yml

```diff
@@ -34,20 +36,17 @@ services:
             NETBOX_USERNAME: guest
             NETBOX_PASSWORD: guest
         volumes:
-        - $PWD/netbox/netbox/configuration.docker.py:/opt/netbox/netbox/netbox/configuration.py:ro
-        - $PWD/docker/gunicorn_config.py:/opt/netbox/gunicorn_config.py:ro
         - netbox-static-files:/opt/netbox/netbox/static
     nginx:
         image: nginx:1.11.1-alpine
         links:
         - netbox
         container_name: nginx
+        command: nginx -g 'daemon off;' -c /etc/netbox-nginx/nginx.conf
```

Comment:
As the netbox exposes the nginx.conf in a distinct directory (not overwriting the default `/etc/nginx/`), the command has to be adjusted.